common/src/main/java/com/wynntils/handlers/item/ItemHandler.java

```diff
@@ -73,6 +74,9 @@ public void onContainerSetContent(ContainerSetContentEvent.Pre event) {
     }
 
     private void updateAnnotation(ItemStack existingItem, ItemStack newItem) {
+        // These items will already have an annotation, no handling necessary
```

Comment:
I don't think this is needed. Fake item stacks will get an annotation "at birth", and just a few lines below this case will be taken care of.

Reply:
If this return is removed, the annotations get overwritten with WynnItem ones: 
![image](https://user-images.githubusercontent.com/3767283/217773165-fc2ea53c-8249-4ad4-ae86-47df0dcf1562.png)

Reply:
Hm, that looks bad. It is classified as a general, non-specific WynnItem, not a GearItem. Maybe the Fake Item Stacks are improperly setup? I'll check...

Reply:
No, they should have proper annotations. That's weird. 

But let's do it like this. You check in this PR as-is, and then I can go and debug the issue in my own time, to figure out what is going wrong, since this should not have to be here.